.travis.yml

```diff
@@ -23,13 +39,12 @@ notifications:
         - "chat.freenode.net#dokuwiki"
     on_success: change
     on_failure: change
-services:
-  - "mysql"
-  - "postgresql"
```

Comment:
@photodude one question: why do you disable these services globally? are they enabled by default in the non-sudo container?